lfs/ssh.go

```diff
@@ -22,7 +23,11 @@ func sshAuthenticate(endpoint Endpoint, operation, oid string) (sshAuthResponse,
 
 	tracerx.Printf("ssh: %s git-lfs-authenticate %s %s %s",
 		endpoint.SshUserAndHost, endpoint.SshPath, operation, oid)
-	cmd := exec.Command("ssh", endpoint.SshUserAndHost,
+	ssh := os.Getenv("GIT_SSH")
+	if len(ssh) == 0 {
+		ssh = "ssh"
+	}
```

Comment:
This is great, but typically we've been putting this stuff in the `*lfs.Configuration` struct so that we only have to check the ENV once per git lfs process.